app/Exceptions/Handler.php

```diff
@@ -51,6 +41,17 @@ public function report(Throwable $exception) {
      * @throws Throwable
      */
     public function render($request, Throwable $exception) {
-        return parent::render($request, $exception);
+        if (!config('app.debug') && !$exception instanceof Referencable) {
+            $exception = new Referencable();
+            Log::error('Reference for above exception: '.$exception->reference);
```

Comment:
Das habe ich nicht verstanden. Wie bekommt `new Referencable()` die Root Cause? Und wie kommt die "above exception" in das Logfile? Fragen über Fragen..

Reply:
Der Laravel Exceptionhandler loggt __immer__ weg. In dem Fall ist das nur noch die render-Methode, die selbst nichts mehr loggt. Und scheinbar löst der Handler nochmal ne random Exception aus an der Stelle, wenn es nicht im Debug-Mode ist. (Wenn ich im blade template mit `dd` geloggt hat, wurde irgendeine random xdebug-Exception angezeigt. Im Debug mode wurde die richtige Methode angezeigt)

Idk. Laravel-Magie?

Reply:
Spannend. Ja dann alles klaro :)